packages/spruce-cli/src/generators/AbstractGenerator.ts

```diff
@@ -46,6 +50,13 @@ export default abstract class AbstractGenerator {
 		}
 	}
 
+	/** Delete a directory and all it's contents */
+	public deleteDir(destination: string) {
```

Comment:
Should these file/directory methods go in a utility or be methods of a store?

Reply:
We def need to consolidate read/write logic. thinking a utility makes sense, lets afterparty